client/store.ts

```diff
@@ -36,6 +37,14 @@ const store = new Vuex.Store({
         Vue.delete(state.alerts, message);
       }, 3000);
     },
+    async refreshContacts(state) {
```

Comment:
async changes to the store should be actions rather than mutations

Reply:
I see where this is coming from (I just googled it) but given how the Fritter code didn't make a distinction between mutations and actions (and the async), I'm not sure it's super necessary to change